Documentation/kubernetes.md

```diff
@@ -37,6 +37,71 @@ kubectl create -f rook-operator.yaml
 kubectl -n rook-system get pod
 ```
 
+ 
+#### RBAC for PodSecurityPolicies
+
+If you have activated the [PodSecurityPolicy Admission Controller](https://kubernetes.io/docs/admin/admission-controllers/#podsecuritypolicy) and thus are
+using [PodSecurityPolicies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/), you will require additional 
+(Cluster)RoleBindings for the different ServiceAccounts Rook uses to start the Rook Agent, CEPH Mons and OSD Pods.
+ 
+**Note**: You don not have to perform these steps if you do not have the PodSecurityPolicy Admission Controller activated!  
+
+##### PodSecurityPolicy 
+
+You will probably have a PodSecurityPolicy for elevated access in place, but you need one which allows hostNetwork usage and 
```

Comment:
Could you put "PodSecurityPolicy" and "hostNetwork" in code blocks.
Also you should mention that allowing `hostNetwork` usage is only required when using `hostNetwork: true` in the Cluster CRD in the yaml below as a comment for `hostNetwork` with default `false` is good too.